apps/searchjoin/db.h

```diff
@@ -130,25 +130,25 @@ namespace seqan
 template <>
 struct Fibre<TDbDnaSaSmall, FibreSA>
 {
-    typedef String<Pair<unsigned int, unsigned char, BitPacked<24, 8> >, StringSpec<TDbDnaSaSmall>::Type> Type;
+    typedef String<Pair<unsigned int, unsigned char, BitPacked<24, 8> >, DefaultIndexStringSpec<TDbDnaSaSmall>::Type> Type;
```

Comment:
`DefaultIndexStringSpec` was previously defined?

Reply:
Yes, it was previously defined and marked deprecated in by @smehringer's PR. However since she also replaced the internal use of the function it also meant that downstream users who overloaded this metafunction no longer actually influenced the code in SeqAn. This made it a breaking change.
Additionally, due to definition/instantiation/specialization times of the templates it is not possible to just use `StringSpec` instead.

Since I couldn't fix it in a reasonable amount of time in my app, I concluded most of our users wouldn't be able to either.

Reply:
ok